src/test/java/org/broadinstitute/hellbender/utils/smithwaterman/SWPairwiseAlignmentUnitTest.java

```diff
@@ -117,4 +117,119 @@ public void  testForIdenticalAlignmentsWithDifferingFlankLengths() {
         }
     }
 
+    @Test(enabled = true)
+    public void testSubstringMatchSoftclip() {
+        final String match     = "CCCCC";
+        final String reference = "AAA" + match;
+        final String read      = match;
+        final int expectedStart = 3;
+        final String expectedCigar = "5M";
+        final SWPairwiseAlignment sw = new SWPairwiseAlignment(reference.getBytes(), read.getBytes(),
```

Comment:
\[FindBugs\] WARNING: Dm: Found reliance on default encoding in org\.broadinstitute\.hellbender\.utils\.smithwaterman\.SWPairwiseAlignmentUnitTest\.testSubstringMatchSoftclip\(\): String\.getBytes\(\)